Relativity.Audit.SDK.md

```diff
@@ -25,4 +26,4 @@ Lowest Version | Highest Version
 
 Lowest Version | Highest Version
 --- | ---
-21.0.0 | Latest
+22.2.10 | Latest

```

Comment:
The intention of these doc seems to be for the versioned Relativity.Audit.SDK otherwise the information of Relativity.Audit.Services.SDK should have been added when this doc was initially created. 

Do we have the versions of relativity and the compatible versions of Relativity.Audit.Services.SDK documented somewhere else? 
In terms of versions for Relativity and  Relativity.Audit.Services.SDK how far back should we go in this doc?

Reply:
This is the only other place I can find documentation on versions of the audit API 
https://platform.relativity.com/RelativityOne/Content/BD_Audit/Audit_APIs.htm
In that doc this is the only mention of versions 
**RelativityOne 10.2.170.2 and above Relativity.Audit.Services.SDK 15.2.3**


RelativityOne 10.2.170.2 and above | Relativity.Audit.Services.SDK 15.2.3
-- | --



RelativityOne 10.2.170.2 and above | Relativity.Audit.Services.SDK 15.2.3
-- | --



RelativityOne 10.2.170.2 and above | Relativity.Audit.Services.SDK 15.2.3
-- | --

Reply:
This repo is meant to host compatibility information for newer packages which uses the Publish To NuGet pipeline to publish packages. Relativity.Audit.Services.SDK was previously published manually and hence it's not included in this repo.